### PR TITLE
Dockerfile fixes

### PR DESCRIPTION
**Pull request by elizlee**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ RUN apt-get update --fix-missing && apt-get install -y \
     libboost-all-dev \
     software-properties-common \
     unzip \
+    vim \
```
Comment (joelb-git) [inline Dockerfile:22]: Thank you!

File: Dockerfile [hunk 2]
```diff
@@ -79,6 +87,10 @@ RUN wget https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transfor
     mv stsb-roberta-base /cdse-covid/wikidata_linker/sent_model/ && \
     rm stsb-roberta-base.zip
 
+# Create torch env
+ENV TORCH_HOME=/cdse-covid/phase3_test/.cache/torch
```
[the remark this answers is not in the document]
Comment (elizlee) [reply Dockerfile:91]: Thank you for reminding me -- I'll add instructions for those

[the remark this answers is not in the document]
Comment (joelb-git) [reply Dockerfile:91]: Can it be done inside the Dockerfile?  Would make things easier for users.

File: Dockerfile [hunk 3]
```diff
@@ -63,21 +69,32 @@ ENV PYTHONPATH /cdse-covid
...
+WORKDIR $TORCH_HOME/hub
+WORKDIR /
+
 # Download sentence model weights
 RUN wget https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/stsb-roberta-base.zip && \
     unzip stsb-roberta-base.zip -d stsb-roberta-base && \
     mv stsb-roberta-base /cdse-covid/wikidata_linker/sent_model/ && \
-    rm stsb-roberta-base.zip
+    rm stsb-roberta-base.zip && \
+    wget https://github.com/pytorch/fairseq/archive/main.zip && \
```
Comment (joelb-git) [inline Dockerfile:94]: Could you say what this is?  I can't find much from googling.  Maybe add a comment to the Dockerfile.

**Comment by joelb-git** [comment]
I was able to run the full pegasus pipeline that uses the docker image produced from this Dockerfile.
So I think it can be merged.  There is a question on the `python_tap_amr_parsing` step where it
took maybe an hour for 5 documents.  The whole pipeline on 5 docs too 132m.  It's possible
amr parsing was not properly using a gpu, but that's just speculation at this point.

**Review by joelb-git — APPROVED** [review]

**Pull request merged**